server/jwt/jwt_utils.go

```diff
@@ -0,0 +1,22 @@
+package jwt
+
+import "strings"
+
+func Contains(s []string, searchterm string) bool {
```

Comment:
this could be replaced by using `ContainsString` from `utils/slice/slice.go`.

Reply:
done!